### ISSUE TITLE
[Feature request] Make full use of five-button mouse

### ISSUE THREAD

**Issue by NomadicVoxel** [Genymobile/scrcpy#2110]
- [x] I have checked that a similar [feature request](https://github.com/Genymobile/scrcpy/issues?q=is%3Aopen+is%3Aissue+label%3A%22feature+request%22) does not already exist.

It would be awesome if a five-button mouse could be fully utilized. (specifically, so the back button on the mouse is the back button in android, not right-click)

**Is your feature request related to a problem? Please describe.**
Honestly, the biggest issue I'm dealing with when using this app is forgetting that right-click does NOT mean the context menu like literally everything else. It quite often erases work, too, if the app leaves the form you were entering data into. 
I get it if you've only got a two-button or three-button mouse, but it would be SO useful to have an option to use the two other buttons a five-button mouse provides for this use.

**Describe the solution you'd like**
I don't know if software can detect which kind of mouse is available, so perhaps this should be set up as an option. 
But how I imagine this working, if enabled, the five-button mouse config would remap to something like this: 
- Left-click is a tap event as usual
- Right-click is either a tap event, or if possible triggers the context menu like a long-tap. (it's fine if it's just a tap event, this one's low priority) 
- Middle click can stay as returning to the app launcher. 
- Mouse4, forward button, can view Open Apps (whatever that's properly called. App Switcher?)
- Mouse5, back button, _should be the back button_

**Describe alternatives you've considered**
I suppose the other major alternative idea would be to have a way to remap all shortcuts and events... 

Thanks for reading this

**Comment by lordmundi** [comment]
just want to also vote for this.  I came looking for any documentation on how I could use the mouse to open the notification menu.  As I get notifications, trying to click and drag is very awkward.  I see there is a keyboard shortcut, but most of my interaction with scrcpy is with the mouse, so I was hoping I could customize the mouse button mapping somehow to make something open up the notification menu.  Since I have extra buttons on my mouse, using one of the side buttons would be great if it could open up the notification menu. Searching I found this issue and just wanted to voice support.  scrcpy is great!
